nixos/modules/profiles/base.nix

```diff
@@ -45,6 +45,10 @@
   # Include support for various filesystems and tools to create / manipulate them.
   boot.supportedFilesystems =
     [ "btrfs" "cifs" "f2fs" "ntfs" "vfat" "xfs" ] ++
+    # FIXME: Change the test condition to
+    # config.boot.kernelPackages.kernel.kernelAtLeast "6.7"
+    # when bcachefs module no longer defaults to latest kernel
```

Comment:
Why does it needs the latest kernel instead of 6.7?

Reply:
Currently, since Linux 6.12 is not yet the default pkgs.linuxPackages, bcachefs module will [set the default kernel to pkgs.linuxPackages_latest](https://github.com/NixOS/nixpkgs/blob/50bb98b521635b4c227933eefe30c6d800ae719d/nixos/modules/tasks/filesystems/bcachefs.nix#L140). So right now we only enable it if the user is actually building an image with latest kernel.